README.md

```diff
@@ -19,15 +19,12 @@ OpenSSL (ssl, crypto)  | OpenSSL License
 libxml2                | MIT
 ldoc                   | MIT/X11
 
-For ATF usage:
-```sudo apt-get install liblua5.2 libxml2 lua-lpeg```
-
 ## Get source code:
 ```
-$ git clone https://github.com/smartdevicelink/sdl_atf
-$ cd sdl_atf
-$ git submodule init
-$ git submodule update
+git clone https://github.com/smartdevicelink/sdl_atf
+cd sdl_atf
+git submodule init
+git submodule update
 ```
 ## Compilation:
 **1** Install 3d-parties developers libraries
```

Comment:
I'll note that there are a few package I needed to install which are missing from this list:
```
sudo apt-get install libqt5websockets5 libqt5websockets5-dev 
```

Reply:
@jacobkeeler Yes, the packages you mentioned are applicable for Ubuntu `18.04`. 
As for `16.04` it's enough to have the ones that were described already.
I would suppose this update is not required for current proposal.
However we going to update this section in scope of `WebEngine` proposal since this new feature requires newer version of QT - `5.9`

Reply:
I would think that it should be included, since the default (and recommended) environment for the project is Ubuntu `18.04`.

Reply:
@jacobkeeler Please find update in ed2a020